src/devicelock/devicelock.cpp

```diff
@@ -199,7 +199,9 @@ bool DeviceLock::runPlugin(const QStringList &args)
     qWarning() << process.readAllStandardError();
 #endif
 
-    return process.exitCode() == 0;
+    if (process.exitCode() == 0) return OK;
```

Comment:
current enum order is there because of keeping compatibility with jolla-home and settings-system. That way people don't get locked out of their devices if they only update lipstick and somehow forget to update jolla-home and settings-system at the same time. (as those projects have hard coded in some places that OK == 1), so it would be safer to keep compatibility and gradually create pull request to switch hardcoded values to use the enum.

And anyways this pull request is currently on hold as devicelock rewrite study might change the way how the pass code age is handled.